float32 x_trans
float32 y_trans
float32 z_trans
string marker_name
string subject_name
string segment_name
bool occluded